api/services/gameService.js

```diff
@@ -232,6 +232,11 @@ module.exports = {
   },
 
   dealCards: function (game, gameUpdates) {
+    // Use gamestate api if feature flag is set, otherwise legacy
+    if (process.env.VITE_USE_GAMESTATE_API) {
+      return sails.helpers.gamestate.dealCards(game);
+    }
+
```

Comment:
Dynamically use either gamestate api version or legacy based on feature flag. When we finalize the project we can/should delete this function altogether and directly call the gamestate helper in the consumers